.github/workflows/main.yml

```diff
@@ -54,14 +71,15 @@ jobs:
           if [[ ${{ matrix.torch-version }} != "1.0.1" ]]; then pip install mypy==0.790; fi
           pip install pytest
           pip install --progress-bar off coverage codecov
+          pip install --progress-bar off --upgrade numpy==${{ matrix.numpy-version }}
       - name: Linting pylint and flake8 if torch version is 1.4.0
         run: |
           if [[ ${{ matrix.torch-version }} == "1.4.0" ]]; then pylint texar/ examples/; fi
           if [[ ${{ matrix.torch-version }} == "1.4.0" ]]; then flake8 texar/ examples/; fi
-      - name: Linting with mypy if torch versoin is not 1.0.1
+      - name: Linting with mypy if torch versoin is not 1.0.1 and numpy>=1.20
```

Comment:
maybe ==1.21`?

Reply:
now it tests numpy 1.20 and 1.21. do we want to test 1.21 only?

Reply:
I think type checking against one version should be good enough. In the current setting, newer version of numpy (say 1.3) may introduce different types which can break some tests)

Reply:
got it! updated.